src/plugins/dashboard/public/saved_dashboards/saved_dashboard.ts

```diff
@@ -20,27 +20,34 @@ import { extractReferences, injectReferences } from '../../common/saved_dashboar
 
 import { DashboardOptions } from '../types';
 
-export interface DashboardSavedObject extends SavedObject {
+export interface DashboardAttributes {
```

Comment:
I am very confused by interfaces of the SavedObject in Dashboard. This seems to be legacy... but working with the current interface

If I am not wrong a saved object interface should be like this:

```typescript
interface MySavedObject {
  id : string // core
  updatedAt: string // core
  ... // all other root fields from core
  attributes: {
    ... // this is where a saved object type can declare its own fields
  }
}
```